cli/src/main.rs

```diff
@@ -144,6 +157,105 @@ async fn poll_server_listening() -> Result<(), DynError> {
     }
 }
 
+async fn trigger(domain: &String, http_port: &u16, rpc_port: &u16, open_browser: &bool) {
+    let (client, wait) = start_server(domain, http_port, rpc_port, open_browser);
+
+    // Spin off child process to make sure it can make process on its own
+    // while we read its output
+
+    let mut process = client.process;
+
+    // todo tokio select on this
+    let task_handle = tokio::spawn(async move {
```

Comment:
If this `task_handle` returns we need to abort. it's indicative the task shut down.

Reply:
I suspect this solves another possible scenario I still needed to test:

running `cargo run stop` while the development_server is running should terminate the development_server process

🙌